src/core/Akka/Event/Logging.cs

```diff
@@ -270,7 +270,7 @@ public static LogLevel LogLevelFor(string logLevel)
         {
             if (!string.IsNullOrEmpty(logLevel))
             {
-                logLevel = logLevel.ToUpper();
+                logLevel = logLevel.ToUpper(new System.Globalization.CultureInfo("en-EN"));
```

Comment:
Can you test `ToUpperInvariant` instead?

Reply:
Yes, I just tested it with `ToUpperInvariant ` and worked fine.